pkg/rancher-desktop/main/commandServer/httpCommandServer.ts

```diff
@@ -80,6 +81,14 @@ export class HttpCommandServer {
         '/v1/extensions/uninstall': [1, this.uninstallExtension],
       },
     } as const,
+    {
+      get:  { '/v1/snapshots': [0, this.listSnapshots] },
+      post: {
+        '/v1/snapshot':         [0, this.createSnapshot],
+        '/v1/snapshot/restore': [0, this.restoreSnapshot],
+      },
+      delete: { '/v1/snapshot': [0, this.deleteSnapshot] },
```

Comment:
And this should be `DELETE /v1/snapshots` because you're modifying the list of snapshots